src/Slider/Slider.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import Slider from "./Slider";
+
+/**
+ * Tests
+ */
+describe("Slider", () => {
+  test("can renders without crashing", () => {
```

Comment:
This sounds a bit brutal 'without crashing'. Maybe something a bit more like 'can render with no props'.